recipes/clara/all/conanfile.py

```diff
@@ -1,29 +1,33 @@
 import os
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
 
-required_conan_version = ">=1.28.0"
+required_conan_version = ">=1.46.0"
 
 class ClaraConan(ConanFile):
     name = "clara"
     description = "A simple to use, composable, command line parser for C++ 11 and beyond"
     homepage = "https://github.com/catchorg/Clara"
-    topics = ("conan", "clara", "cli", "cpp11", "command-parser")
+    topics = ("clara", "cli", "cpp11", "command-parser")
```

Comment:
we dont need the project name in the topics - it does not improve the search

```suggestion
    topics = ("cli", "cpp11", "command-parser")
```